starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,48 @@
+h1 {
+    color: white;
+    text-align: center;
+    background: url(/Users/DanielAutrey/Desktop/ironhack/Individual/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg) 0 0 no-repeat;
```

Comment:
Please keep on your mind that only on your machine this image exists on this path. We should do something like
```
background: url("images/someImage.jpg");
```